Util/FormBuilderIterator.php

```diff
@@ -65,12 +65,7 @@ public function __construct(FormBuilderInterface $formBuilder, $prefix = false)
      */
     private static function getKeys(FormBuilderInterface $formBuilder)
     {
-        if (!self::$reflection) {
-            self::$reflection = new \ReflectionProperty(get_class($formBuilder), 'children');
-            self::$reflection->setAccessible(true);
-        }
-
-        return array_keys(self::$reflection->getValue($formBuilder));
+        return array_keys($formBuilder->all());
```

Comment:
Try to keep the commit message subject under 50 characters. The commit message body can be as long as you want. See that ugly `…s over reflection` above? Try to get rid of it if you can.